src/main/features/claim/helpers/featuresBuilder.ts

```diff
@@ -29,7 +29,12 @@ export class FeaturesBuilder {
       }
     }
 
-    if (draft.document.amount.totalAmount() <= ONLINE_DQ_THRESHOLD) {
+    if (draft.document.amount.totalAmount() <= JUDGE_PILOT_THRESHOLD) {
```

Comment:
this should only be added if it is above the legal advisor pilot and below the threshold